source/ways-of-working/environments/aks-auto-shutdown.html.md.erb

```diff
@@ -0,0 +1,57 @@
+---
+title: AKS Auto Shutdown
+last_reviewed_on: 2023-02-07
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+### Schedule
+
+See below the list of AKS clusters that have auto-shutdown enabled alongside their schedule:
+
+| Shared Services Environment | CFT Environment | Schedule |
+|-|-|-|
+| Demo | Demo | 6:30pm |
+| ITHC | ITHC | 6:30pm |
+| Test | Perftest | 6:30pm |
+| PTLSBOX | PTLSBOX | 6:30pm |
+| Sandbox | Sandbox | 6:30pm |
+
+
+### How to start and stop the cluster from pipeline
+
+If required, an AKS cluster can be started or stopped using a github action on the aks-auto-shutdown repository.
+
+#### Start a cluster
```

Comment:
This will start all environments won't it?

Don't we need to add a parameter that they can optionally pass or a new workflow just for manually starting / stopping an individual environment?

Reply:
I would say so yes, should we do that as part of this ticket? Think it would require taking another look at the code.

Reply:
yes I think it's worth doing it quickly as we don't want someone starting all environments

Reply:
Okay cool I'll take a look :)